about.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" 
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" 
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <title>WebTekhy | About page</title>
+</head>
+<body id="hideOverflow">
+    <header class="header" id="header">
+        <div class="logo">
+            <a href="index.html"><img class="logo-img" src="images/webtekhy-icon.png" alt="webtekhy icon"></a>
+        </div>
+        <nav id="menu_bar" class="navbars">
+            <i id="menuBarImg" class="fa fa-bars" aria-hidden="true"></i>
+            <div id="close_bar" class="close-menu">
+                <i id="close_menu" class="close" >&times;</i>  
```

Comment:
- [ ] Good job adding the top black sub-menu to the home page requested by the previous reviewer, But it should also be available for the about page as it is on the [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) design. Please make sure you add that sub-menu to the about page as well.